R/nested.R

```diff
@@ -24,12 +24,13 @@
 #'     - Function calls
 #'     - Function definitions
 #' - Remove `includeText = TRUE`
+#' @importFrom purrr rerun
 compute_parse_data_nested <- function(text) {
   parsed <- parse(text = text, keep.source = TRUE)
   parse_data <- tbl_df(utils::getParseData(parsed, includeText = TRUE))
   pd_nested <-
     parse_data %>%
-    mutate_(child = ~rep(list(NULL), length(text))) %>%
+    mutate_(child = ~rerun(length(text), tibble())) %>%
```

Comment:
An empty tibble will cost us up to 700 bytes per terminal, whereas `NULL` costs nothing extra. I'm not sure how much this is relative to the object sizes we're already handling, but we should be aware of this.

Could we perhaps use an adapted version of `bind_rows()` that returns `NULL` iff the result has zero rows?